src/setup/anvil/templates/anvil-deploy-dependencies.sh.njk

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
```

Comment:
let's polish all these templates (remove comments, using quotes where needed, and use some prevention measures like
```
Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```

Reply:
Also, let's remove the API keys if not needed